engine/src/main/scala/cromwell/server/WorkflowManagerSystem.scala

```diff
@@ -24,7 +24,10 @@ trait WorkflowManagerSystem {
   def allowedBackends: List[String] = ConfigFactory.load.getConfig("backend").getStringList("backendsAllowed").toList
   def defaultBackend: String = ConfigFactory.load.getConfig("backend").getString("defaultBackend")
 
+  val isServerMode = false
```

Comment:
Maybe make this a not implemented `def` and force implementors to give a value for it ? Or is there a good reason `false` should be the default ?